docs/section19/19-7.md

```diff
@@ -0,0 +1,87 @@
+        - Get the analogous zone object from the proposed RMI (if this is not strictly the correct syntax please advise Weili): `zone_p = get_component_by_id(P_RMI,match_data_element(P_RMI,Zone,zone.id))`    
+        - Add each zone's OA CFM schedule to the list of zone OA CFM schedules for the proposed: `zone_OA_CFM_list_of_schedules_p[counter] = get_min_OA_CFM_sch_zone(P_RMI, zone_p.id)`  
+
+        - For each terminal unit serving the zone, check if DCV was modeled: `for terminal in zone.terminals:`  
+            - Check if DCV was modeled: `if terminal.has_demand_control_ventilation == true: was_DCV_modeled_baseline = true`  
+        
+        - Check if the air distribution effectiveness is greater than 1 in the proposed: `if zone_p.air_distribution_effectiveness >1: zone_air_distribution_effectiveness_greater_than_1 = true`  
+        
+        - For each terminal unit serving the zone in the proposed, check if DCV was modeled: `for terminal in zone_p.terminals:`  
+            - Check if DCV was modeled: `if terminal.has_demand_control_ventilation == true: was_DCV_modeled_proposed = true`  
+            - Check if the hvac system serving the terminal unit has dcv modeled (weili may have to include some logic to get the hvac object from the id): `if terminal.served_by_heating_ventilating_air_conditioning_system.fan_system.demand_control_ventilation_control == true: was_DCV_modeled_proposed`  
+        
+        - Add to counter: `counter = counter + 1`  
+    
+    - Create aggregated OA CFM schedule for the baseline for the zones served by the hvac system: `aggregated_min_OA_schedule_across_zones_b = aggregate_min_OA_schedule_across_zones(B_RMI,zone_OA_CFM_list_of_schedules_b)`  
+    - Create aggregated OA CFM schedule for the proposed for the zones served by the baseline hvac system: `aggregated_min_OA_schedule_across_zones_p = aggregate_min_OA_schedule_across_zones(P_RMI,zone_OA_CFM_list_of_schedules_p)`  
+
+    - Reset OA_CFM_schedules_match boolean variable to True: `OA_CFM_schedules_match = True`  
+    - Check if schedules mismatch for any hours of the year, if so set boolean to false: `for p to Range(8760):`  
+        - Check if the baseline and proposed schedules mismatch: `if aggregated_min_OA_schedule_across_zones_b[p] != aggregated_min_OA_schedule_across_zones_p[p]: OA_CFM_schedule_match = false`   
+
+    - Sum OA CFM across baseline schedule to get the total for comparison purposes in the rule assertions: `modeled_baseline_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_b)`  
+    - Sum OA CFM across proposed schedule to get the total for comparison purposes in the rule assertions: `modeled_proposed_total_zone_min_OA_CFM = sum(aggregated_min_OA_schedule_across_zones_p)`  
+
+
+
+    - **Rule Assertion:** 
+    - Case 1: If the modeled baseline and proposed minimum OA cfm values are identical then pass: `if OA_CFM_schedule_match == true: outcome = "PASS"`  
```

Comment:
What if spaces are over-ventilated and proposed should actually be less than baseline?

Reply:
Proposed less than baseline?

Reply:
Done